features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/telemetry/internal/TelemetryEventHandler.kt

```diff
@@ -158,6 +158,10 @@ internal class TelemetryEventHandler(
         return true
     }
 
+    private fun isLog(event: InternalTelemetryEvent): Boolean {
+        return event is InternalTelemetryEvent.Log
+    }
```

Comment:
/improvement

this can be extension method.

Reply:
we tend no to use extension methods in Kotlin unless is being re - used in other classes. Extension method tend to be heavy at the bytecode level.